### PR TITLE
Allow latest pkg:material_color_utilities

### PR DESCRIPTION
**Pull request by kevmoo**
* Allow latest pkg:material_color_utilities
* Bump other dependencies to their latest - including pkg:web

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Review by guidezpl — DISMISSED** [review]
`flutter update-packages` is needed, right?

**Comment by christopherfujino** [comment]
friendly ping @kevmoo

**Comment by kevmoo** [comment]
I'd LOVE someone to pick this up! I worry I'll waste more time w/ the reviewers that someone who knows how to hold everything right!

**Comment by christopherfujino** [comment]
> I'd LOVE someone to pick this up! I worry I'll waste more time w/ the reviewers that someone who knows how to hold everything right!

sounds good, moving this to draft in the meantime to get it off our review queue.

**Review by christopherfujino — APPROVED** [review]
LGTM

File: packages/flutter_tools/lib/src/commands/update_packages.dart [hunk 1]
```diff
@@ -33,8 +33,8 @@ const Map<String, String> kManuallyPinnedDependencies = <String, String>{
   'flutter_template_images': '4.2.0', // Must always exactly match flutter_tools template.
   'video_player': '2.2.11',
   // Keep pinned to latest until 1.0.0.
-  'material_color_utilities': '0.5.0',
-  // https://github.com/flutter/flutter/issues/115660
+  'material_color_utilities': '0.8.0',
+  // https://github.com/flutter/flutter/issues/111304
```
Comment (christopherfujino) [inline packages/flutter_tools/lib/src/commands/update_packages.dart:37]: oh wait, this looks wrong?

**Review by christopherfujino — DISMISSED** [review]

File: packages/flutter_tools/lib/src/commands/update_packages.dart (hunk 1 above)
Comment (christopherfujino) [reply packages/flutter_tools/lib/src/commands/update_packages.dart:37]: I'm guessing a the diff resolution algorithm picked the wrong one

Comment (kevmoo) [reply packages/flutter_tools/lib/src/commands/update_packages.dart:37]: yup! will fix!

**Review by christopherfujino — APPROVED** [review]
ship it again!

**Comment by jperezr21** [comment]
Hi! This was merged on October, but the latest stable version which was released last week is still using meta 1.10.0. I'm getting the following error due to that:
```
Because every version of flutter_test from sdk depends on meta 1.10.0 and app depends on meta ^1.11.0, flutter_test from sdk is forbidden.
So, because app depends on flutter_test from sdk, version solving failed.
```

**Comment by christopherfujino** [comment]
> Hi! This was merged on October, but the latest stable version which was released last week is still using meta 1.10.0. I'm getting the following error due to that:
> 
> ```
> Because every version of flutter_test from sdk depends on meta 1.10.0 and app depends on meta ^1.11.0, flutter_test from sdk is forbidden.
> So, because app depends on flutter_test from sdk, version solving failed.
> ```

I would change your app constraint to be `meta: ^1.10.0`

**Comment by jperezr21** [comment]
I'm actually trying to depend on analyzer 6.3.0, which depends on meta 1.11.0. This is the original error I was getting:
```
[WARNING] riverpod_generator on lib/contexts/auth/application/auth_controller.dart:
Your current `analyzer` version may not fully support your current SDK version.

Analyzer language version: 3.1.0
SDK language version: 3.2.0

Please update to the latest `analyzer` version (6.3.0) by running
`flutter packages upgrade`.

If you are not getting the latest version by running the above command, you
can try adding a constraint like the following to your pubspec to start
diagnosing why you can't get the latest version:

dev_dependencies:
  analyzer: ^6.3.0
```

**Pull request merged**